lib/web/JsonpChunkLoadingRuntimeModule.js

```diff
@@ -168,9 +168,21 @@ class JsonpChunkLoadingRuntimeModule extends RuntimeModule {
 													`var url = ${RuntimeGlobals.publicPath} + ${RuntimeGlobals.getChunkScriptFilename}(chunkId);`,
 													"// create error before stack unwound to get useful stacktrace later",
 													"var error = new Error();",
+													`var webpackJsonpRuntimeErrorCallback = ${runtimeTemplate.basicFunction(
+														"event",
+														[
+															"if (event.filename === url) {",
+															Template.indent([
+																"error.evaluationError = event.message;"
+															]),
+															"}"
+														]
+													)}`,
+													`window.addEventListener("error", webpackJsonpRuntimeErrorCallback)`,
```

Comment:
This is real not good, jsonp can be used in workers and there is not window

Reply:
i know hehe https://github.com/webpack/webpack/pull/17273#issuecomment-1565237715

if the approach it's ok i will continue working on this direction, i saw there is a helper for global around

Reply:
Hi @alexander-akait  ! 
sorry for the delay (work and some holidays). I have push some changes:

- I'm using now RuntimeGlobals.global instead of window, although the error event is only available on window so web workers won't report the error this way, anyway i have taken a look and i would say errors for web workers should be reported in application code as there is no load event (it would be possible to post a message to simlulate load event but this will require to implement some logic around the web worker in webpack's side)
- i have added a test case for this feature and added methods to the global context as current "window"does not have event listeners methods. I must say test execution is really heavy on resources (i executed test script and left to do other things meanwhile and my computer was restarted 🥲 ). To be able to execute them in local i put a limit on workers and concurrency for jest and had some errors regarding timeout and regex  backtracking but those test passed on the current checks so i suppose it is ok, just to know how do you execute all test suites in local?

Let me know if further changes are required